### PR TITLE
add utility `add_simple_test`

### PR DESCRIPTION
**Pull request by michaellilltokiwa**
Using `make add_simple_test` will create a new template test in tests.

### CODE REVIEW

File: lib/String.fz [hunk 1]
```diff
@@ -39,6 +39,9 @@ public String ref : property.equatable, property.hashable, property.orderable is
   # returns true if string is empty or contains whitespace only
   public is_blank => utf8 ∀ u -> u.is_ascii_white_space
 
+  # returns true if string contains whitespace
+  public contains_whitespace => utf8 ∃ u -> u.is_ascii_white_space
+
```
Comment (fridis) [inline lib/String.fz:44]: I think this is unrelated to the PR

File: lib/io/file.fz [hunk 2]
```diff
@@ -26,3 +26,10 @@
 # file -- a unit type that contains effects related to file I/O
 #
 public file is
+
+
+  # Does a file or directory at path exist?
+  #
+  public exists(path String) =>
+    (stat path false).ok
```
Comment (fridis) [inline lib/io/file.fz:34]: I think this is unrelated to the PR

**Review by fridis — CHANGES_REQUESTED** [review]
Lookg good, just think there are some unrelated commits that ended up in this PR.

File: lib/String.fz (hunk 1 above)
Comment (michaellilltokiwa) [reply lib/String.fz:44]: @fridis It is related. used in bin/add_simple_test.fz

File: lib/io/file.fz (hunk 2 above)
Comment (michaellilltokiwa) [reply lib/io/file.fz:34]: @fridis also related

**Review by fridis — APPROVED** [review]
